backend/grant/proposal/models.py

```diff
@@ -198,21 +198,18 @@ def validate(proposal):
         if category and category not in CATEGORIES:
             raise ValidationException("Category {} not in {}".format(category, CATEGORIES))
 
-    def validate_publishable(self):
+    def validate_publishable(self, current_user=None):
```

Comment:
In case passing `current_user` around is undesirable, there is the `grant.utils.auth.get_authed_user()` convenience function.

Reply:
Good to know! I still don't think I fully understand who the current_user is in all cases where validate_publishable gets called, since it appears that it could be either the proposer or admin. 

Regardless, the current logic will only provide an additional layer of defense, so there shouldn't be a regression here either way.

Reply:
Ah, I missed the admin case! 

This feels like its mixing authentication with validation a little bit. Would modifying the auth wrapper func like `@requires_auth(validated=True)` make sense?

Reply:
👍 Working on this now